lib/utils/utils.ts

```diff
@@ -142,3 +142,33 @@ export const ms = (): number => {
 export const derivePairId = (pair: Pair) => {
   return `${pair.baseCurrency}/${pair.quoteCurrency}`;
 };
+
+/**
+ * A simplified copy of lodash's isPlainObject;
+ *
+ * Checks is the object is a plain object;
+ * A plain object's;
```

Comment:
I think more clear to say "A plain object is:"

Otherwise great comment block!

Reply:
done